aten/src/ATen/templates/FunctionalInverses.h

```diff
@@ -22,6 +22,10 @@ struct FunctionalInverses {
 
 ${view_inverse_declarations}
 
+// NB: These are not generated! They're manually implemented in the template.
+static at::Tensor chunk_copy_inverse(const at::Tensor & base, const at::Tensor & mutated_view, InverseReturnMode inverse_return_mode, int64_t mutated_view_idx, int chunks, int dim);
+static at::Tensor narrow_copy_inverse(const at::Tensor & base, const at::Tensor & mutated_view, InverseReturnMode inverse_return_mode, int dim, c10::SymInt start, c10::SymInt length);
+
```

Comment:
> It would be nice though if these are generated as long as ADInplaceOrView kernels are generated.

I agree! But that requires some amount of codegen rework. Currently:
* Functionalization codegen operates on `NativeFunctionsViewGroup`s.
    * There are no `NativeFunctionsViewGroup`s for narrow (see comment [here](https://github.com/pytorch/pytorch/blob/70f3a530d7290d9eb010ad789787bec4dcbfbeb9/torchgen/model.py#L2583-L2585)) or chunk, which is why they have to be defined here.
* ADInplaceOrViewTypeX codegen operates on `NativeFunctionWithDifferentiabilityInfo`s.

It might be possible to clean this up. I'll add a TODO here for re-examining codegen

Reply:
Sounds good!